Update/onik_tips_16.txt

```diff
@@ -368,6 +365,7 @@ void main()
 	ClearMessage();
 
 //「.........なんてわけはありませんねぇ。＠
+	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
```

Comment:
Get rid of the `if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_Continue); }` below.